asn1ate/pyasn1gen.py

```diff
@@ -567,23 +574,37 @@ def _sanitize_identifier(name):
     return name
 
 
+def _sanitize_module(name):
+    """ Sanitize ASN.1 module identifiers so that they're PEP8 compliant identifiers.
+    """
+    return _sanitize_identifier(name).lower()
+
 # Simplistic command-line driver
-def main(args):
-    with open(args[0]) as f:
-        asn1def = f.read()
+def main():
+    arg_parser = argparse.ArgumentParser(description='Generate Python classes from an ASN1 definition file. Output to stdout by default.')
+    arg_parser.add_argument('filename', metavar='filename', type=argparse.FileType('r'),
+                        help='the ASN1 file to process')
+    arg_parser.add_argument('--split', action='store_true',
+                        help='output multiple modules to separate files')
```

Comment:
Indentation looks funny here, can you align `help` with the arguments on the preceding line?